### PR TITLE
Update retention-tags-and-policies.md

### PR DESCRIPTION
**Pull request by coffeemade**
Add this From a retention perspective, the Primary Mailbox and Online Archive should not be looked at as separate entities.  Retention settings are applied to the Primary Mailbox, by design, extend to the Online Archive. The Online Archive is an Extension of the Primary Mailbox.  
Remove this:
Archive mailboxes don't have a separate retention policy. The same retention policy is applied to the primary and archive mailbox.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/security-and-compliance/messaging-records-management/retention-tags-and-policies.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ A retention policy can have the following retention tags:
 > [!NOTE]
 > Although a retention policy doesn't need to have any retention tags linked to it, we don't recommend using this scenario. If mailboxes with retention policies don't have retention tags linked to them, this may cause mailbox items to never expire.
 
-A retention policy can contain both archive tags (tags that move items to the personal archive mailbox) and deletion tags (tags that delete items). A mailbox item can also have both types of tags applied. Archive mailboxes don't have a separate retention policy. The same retention policy is applied to the primary and archive mailbox.
+A retention policy can contain both archive tags (tags that move items to the personal archive mailbox) and deletion tags (tags that delete items). A mailbox item can also have both types of tags applied.  From a retention perspective, the Primary Mailbox and Online Archive should not be looked at as separate entities.  Retention settings are applied to the Primary Mailbox, by design, extend to the Online Archive. The Online Archive is an Extension of the Primary Mailbox.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/security-and-compliance/messaging-records-management/retention-tags-and-policies.md:128]:
```suggestion
A retention policy can contain both archive tags (tags that move items to the personal archive mailbox) and deletion tags (tags that delete items). A mailbox item can also have both types of tags applied.  From a retention perspective, the primary mailbox and online archive should not be looked at as separate entities.  Retention settings are applied to the primary mailbox and by design, extend to the online archive. The online archive is an extension of the primary mailbox.  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @coffeemade, please commit my suggested changes if you agree with them. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by markjjo** [comment]
@coffeemade PR merged. Should be live by end of day.

**Pull request merged**